Makefile

```diff
@@ -1,5 +1,5 @@
 GO := go
-GO_BUILD = CGO_ENABLED=0 $(GO) build
+GO_BUILD = CGO_ENABLED=1 $(GO) build
```

Comment:
Shall we add `-ldflags "-linkmode 'external' -extldflags '-static'"` to get a statically linked exec?

Reply:
It's in the `libpcap/compile.go`, so `go build` without ldflags can generate statically linked elf:

```go
/*
#cgo LDFLAGS: -L/usr/local/lib -lpcap -static
#include <stdlib.h>
#include <pcap.h>
*/
import "C"
```

Or it's better if we move the flags to `go build`?

Reply:
TIL that `-static` in `#cgo` forces building a statically linked go exec. It's fine to keep it here.